cla_public/templates/base.html

```diff
@@ -113,6 +113,23 @@
 {% block javascripts %}
   <script>
 
+    $(document).ready(function(){
+      /*COVID-19: This function is a stopgap for emergency opening times*/
+      $("#field-thirdparty-time-day, #field-callback-time-day")
+        .find("option:contains(Saturday), option:contains(Sunday)")
+        .remove();
+      $("#thirdparty-time-time_in_day, #thirdparty-time-time_today, #callback-time-time_in_day, #callback-time-time_today")
+        .find("option:contains(7:30 PM), option:contains(6:30 PM), option:contains(5:30 PM)")
+        .remove();
+      var d = new Date();
+      var n = d.getDay();
+      if (n > 5) {
```

Comment:
I also wanted to quickly point out here Malcolm.

Sunday is given the number 0, so this would work for Saturday, but Sunday would not be counted. Wouldn't this need to be changed to 

> n > 5 && n === 0

>

Reply:
Changed - using `||` instead of `&&`